README.md

```diff
@@ -3,4 +3,38 @@
 A curated list of amazingly awesome resources for (not only) LMC developers.
 
 ## Table of Contents
-To Be Done
+- [Development](#development)
+- [Core review](#code-review)
+- [Q&A](#qa)
+- [Git](#git)
+- [Books](#books)
+- [Newsletter](#newsletters)
+
+## Development
+- [@Ocramius - Fluent interfaces are evil](https://ocramius.github.io/blog/fluent-interfaces-are-evil/)
+- Matthias Noback - article series
+    - https://www.ibuildings.nl/blog/2016/01/programming-guidelines-php-developers-part-1-reducing-complexity
```

Comment:
Links are broken, also we can show add here their title via markdown, to not show just the URL.

Proper links:
- https://www.ibuildings.nl/blog/2016/01/programming-guidelines-part-1-reducing-complexity
- https://www.ibuildings.nl/blog/2016/01/programming-guidelines-part-2-getting-rid-null
Third one is OK :)